host-provision.yml

```diff
@@ -0,0 +1,64 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+# Assumption: 
+# First time connection is not through public private key pair
+# FIrst time connection is always through customeradmin credentials password authentication
```

Comment:
Can "custoemradmin" be parameterized?  No need for a command-line parameter, but something in group_vars/all.yml would help in case there's a case when the use is named something else.

Reply:
Done.